src/core/server/plugins/plugin.ts

```diff
@@ -45,6 +45,8 @@ import {
 } from './types';
 import { CoreSetup, CoreStart } from '..';
 
+const { join } = posix;
```

Comment:
why not import join directly?

Reply:
Even though the syntax looks like destructuring, they are in fact named imports. Hence something like `import { posix: { join } }` wouldn't work.